__tests__/functional/register.test.js

```diff
@@ -0,0 +1,123 @@
+import AppRunner from "./utils/AppRunner.js";
+import {
+    expectNoErrors,
+    expectAnyError,
+    expectAnyErrorMessageToBe,
+} from "./utils/matchers.js";
+
+let SUT = new AppRunner(["users"]); //collection 'users' will be emptied after every test
+
+const registerQuery = `mutation registerUser($username: String!, $password: String!, $email: String!) {
+				register(username: $username, password: $password, email: $email) {
+				user_id
+				username
+			}}`;
+
+function createRegisterUserMsg() {
+    return {
+        query: registerQuery,
+        operationName: "registerUser",
+        variables: {
+            username: "test_user",
+            email: "grzesiekkasprzak", //currently this email is accepted!
+            password: "qwerty",
+        },
+    };
+}
+
+describe("register users", () => {
+    const registerUser = createRegisterUserMsg();
```

Comment:
variable names should be nouns
```suggestion
    const registerUserMsg = createRegisterUserMsg();
```
also relevant in all further cases